src/main/webapp/resources/rspace_api_inventory_specs_1_104_0.yaml

```diff
@@ -1843,6 +1843,56 @@ paths:
         '200':
           description: Import results.
 
+  /fieldmark/notebooks:
+    get:
+      summary: >-
+        Return the list of the notebooks for the given user
+      description:
+        Return the list of the notebooks for the given user
```

Comment:
I'd update/expand the summary & description, as it's not obvious from reading that this API lists notebooks stored in Fieldmark, not in RSpace. I'd suggest something like:

```
summary: 
List Fieldmark notebooks that could be imported by the current user

description: 
Return the list of Fieldmark notebooks that are readable by the current RSpace user. 
The pre-requisite is that user enables Fieldmark App on RSpace Apps page, and saves their Fieldmark API token. 
```

Reply:
That makes sense to me, changed accordingly